examples/readout-control-plane-node-density.yaml

```diff
@@ -3,7 +3,7 @@ tests :
     index: {{ es_metadata_index }}
     benchmarkIndex: {{ es_benchmark_index }}
     metadata:
-      ocpVersion: 4.15
```

Comment:
there are multiple other places version is specified in this file, can you make sure to edit all of them

Reply:
Updated as suggested.